handlers.js

```diff
@@ -318,14 +508,330 @@
                 reader.readEntries( onreadsuccess.bind(this),
                                     onreaderr.bind(this))
             }
+            }
```

Comment:
weird indentation

Reply:
Yeah, sorry about that. The syntax got a little messy and not everything was lined up